app/presenters/finder_content_item_presenter.rb

```diff
@@ -50,6 +50,8 @@ def details
       summary: metadata.fetch("summary", nil),
       facets: schema.fetch("facets", []),
     }
+    details_hash[:default_order] = metadata["default_order"] if metadata["default_order"]
```

Comment:
Thanks. I'll look into doing it like that. Quick question: what does the underscore mean in `|_, value|`?